### DISCUSSION TITLE
GitHub-Achievements

### DISCUSSION THREAD

**Discussion by limitless1119** [community/community#146286] [New to GitHub] [answered]
I am a newbie of github. I'd like to get the achievements on github. Specifically YoLo achievement. :)

**Comment by crazybee1001** [comment, marked as answer]
- You can find projects by searching GitHub or visiting sites like Awesome for Beginners.
- Once you find a project, click the Fork button to create a copy in your GitHub account.
- Clone your forked repository to your local machine.
- After making changes, commit and push.
- Go to the original repository and click on Pull Requests.
- Click on New Pull Request, select your forked repository, and submit your pull request and merge.

The End.

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @limitless1119. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
